bindata/manifests/cni-config/sriov-cni-config.yaml

```diff
@@ -35,4 +39,10 @@ spec:
   "link_state":"{{.SriovCniState}}",
 {{- end -}}
   {{.SriovCniIpam}}
-}'
+}
+  {{- if .MetaPluginsConfigured -}}
+  ,
+  {{.MetaPlugins}}
```

Comment:
Do we need curly braces for `{{.MetaPlugins}}`?

Reply:
I guess no:
The idea is to ask the users to fill the array, meaning the expectation is to have them to add the braces (and it's the only way to make it work if they want to chain more than one plugin):
```
{
	// meta plugin A config
},
{
	// meta plugin B config
}
```

Reply:
ack, this is different from how `ipam` is configured, we might need document the string format.

Reply:
Yes, wanted to make sure the approach looked fine first.
Will change this PR with the docs.